packages/web-react/src/components/TooltipModern/demo/TooltipWithFloatingUI.tsx

```diff
@@ -0,0 +1,144 @@
+import { Placement } from '@floating-ui/react';
+import React, { useEffect, useRef, useState } from 'react';
+import { Button } from '../../Button';
+import { Checkbox } from '../../Checkbox';
+import { Grid } from '../../Grid';
+import { Select } from '../../Select';
+import { TooltipModern, TooltipPopover, TooltipTrigger } from '../../Tooltip';
+
+const TooltipWithFloatingUI = () => {
+  const [isOpen, setIsOpen] = useState(true);
+  const [flip, setFlip] = useState(true);
+  const [flipCrossAxis, setFlipCrossAxis] = useState(true);
+  const [shift, setShift] = useState(true);
+  const [size, setSize] = useState(true);
+  const [suggestedPlacement, setSuggestedPlacement] = useState<Placement>('top-start');
+  const [suggestedFallbackPlacement, setSuggestedFallbackPlacement] = useState<Placement[]>([
+    'top',
+    'right',
+    'left',
+    'bottom',
+  ]);
+
+  const suggestedPlacementRef = useRef<HTMLSelectElement>(null);
+  const suggestedFallbackPlacementRef = useRef<HTMLSelectElement>(null);
+
+  const viewportRef = useRef<HTMLDivElement>(null);
+  const contentRef = useRef<HTMLDivElement>(null);
+
+  const handleSuggestedPlacementChange = () => {
+    if (suggestedPlacementRef.current) {
+      setSuggestedPlacement((suggestedPlacementRef.current.value || 'top-start') as Placement);
+    }
+  };
+
+  const handleSuggestedFallbackPlacementChange = () => {
+    if (suggestedFallbackPlacementRef.current) {
+      setSuggestedFallbackPlacement(
+        (suggestedFallbackPlacementRef.current.value.split(',').map((item) => item.trim()) || [
+          'top',
+          'right',
+          'left',
+          'bottom',
+        ]) as Placement[],
+      );
+    }
+  };
+
+  useEffect(() => {
+    const viewport = viewportRef.current;
+    const content = contentRef.current;
+    if (viewport && content) {
+      viewport.scrollLeft = (content.offsetWidth - viewport.offsetWidth) / 2;
+      viewport.scrollTop = (content.offsetHeight - viewport.offsetHeight) / 2;
+    }
+  }, []);
+
+  return (
+    <div className="spirit-feature-tooltip-enable-data-placement">
+      <p>
```

Comment:
Also, this text should be the same in all technologies. If possible.

Reply:
texts unified